File: darts-java/src/main/java/com/stride/scorable/Triple.java
package com.stride.scorable;

public class Triple implements Scorable {
    public static final int MULTIPLIER = 3;

    private int score;

    public Triple(int bedValue) {
        if (bedValue < 1 || bedValue > 20) {
            throw new IllegalArgumentException("Throw bedScore must be in range 1..20");
        }
        this.score = bedValue * MULTIPLIER;
    }

    public int bedScore() {
        return score;
    }
}

File: ruby/lib/darts/bed/triple.rb
module Bed
  class Triple
    MULTIPLIER = 3

    attr_reader :bed_score

    def initialize(bed_value)
      raise ArgumentError.new('Throw score must be in range 1..20') if bed_value < 1 || bed_value > 20
      @bed_score = bed_value * MULTIPLIER
    end
  end
end